.github/workflows/ci-linux_mac.yml

```diff
@@ -110,6 +110,12 @@ jobs:
             core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
             core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
 
+      - name: Install Python
+        if: ${{ !inputs.manylinux }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
```

Comment:
This fixes the macOS CI failures by downgrading from Python 3.13 to 3.12, for which the grpcio dependency of the GCS emulator has pre-built wheels.

Reply:
Let's make CI changes a separate PR from now on. This will be very hard to search for in the future when we'll be wondering why we pinned that python version in the first place and why.

Reply:
Thanks, I'll be more careful in the future. I posted https://github.com/TileDB-Inc/TileDB/commit/4afbe56fed83f59ea7a602186cd3e17bda94578a#commitcomment-148379376 for posterity.

Reply:
Great, thanks! Just sharing the pain from my past mistakes here hoping it might be useful :sweat_smile: